src/tribler-gui/tribler_gui/i18n/README.md

```diff
@@ -0,0 +1,19 @@
+# Internationalization
+
+This app uses QT multi language feature to provider internationalization. 
```

Comment:
provider: A typo?

Reply:
🤔 indeed. This should also state "**the** QT multi language feature".

Reply:
I'll make a quick PR for that, before we forget.